src/lambda_vm/lambda_vm.ml

```diff
@@ -15,7 +15,11 @@ let wrap compute =
   | exception Checks.Out_of_stack -> `Limits Runtime_limits_error.Out_of_stack
   | exception Gas.Out_of_gas -> `Limits Runtime_limits_error.Out_of_gas
 
-module Ir = Ir
+module Ir = struct
+  include Ir
+  let tup_arg_and_storage arg storage =
```

Comment:
I'm not sure about this, I think we could either have type constraints for arg and storage, use the usual type constructors or have custom functions (like `pair`)  for creating this instead.

Reply:
changed to have a module with Value constructors